lib/console.js

```diff
@@ -249,6 +252,74 @@ Console.prototype.groupEnd = function groupEnd() {
     this[kGroupIndent].slice(0, this[kGroupIndent].length - 2);
 };
 
+const untablableTypes = [
```

Comment:
Shall we maybe call it `untableable`?

Reply:
doesn't really make sense, the method below is already named that

Reply:
You misunderstood my comment: I did not mean the whole variable name but only the beginning. And that for all of these variables.

The difference is the `e`.

Reply:
oh. i have no idea. pretty sure `tableable`/`tablable` isn't english anyway so i have no idea where to go with that. i think following other words it would become `tablible`?

Reply:
Maybe @Trott has a opinion about it?